pkg/telemetry/prometheus.go

```diff
@@ -0,0 +1,62 @@
+package telemetry
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/HeavyHorst/remco/pkg/log"
+	"github.com/armon/go-metrics"
+	metricsPrometheus "github.com/armon/go-metrics/prometheus"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+type PrometheusSink struct {
+	Addr       string
+	Expiration int
+
+	httpServer     *http.Server
+	prometheusSink *metricsPrometheus.PrometheusSink
+}
+
+func (p *PrometheusSink) Init() (metrics.MetricSink, error) {
+	if p == nil {
+		return nil, ErrNilConfig
+	}
+
+	handler := http.NewServeMux()
+	handler.Handle("/metrics", promhttp.Handler())
```

Comment:
Could you add a hint to the docs that the metrics are available under addr/metrics? 
I just tried addr and had to look up the handler path in the code.